model/access_test.go

```diff
@@ -29,41 +29,41 @@ func TestAccessIsValid(t *testing.T) {
 	require.NotNil(t, ad.IsValid())
 
 	ad.ClientId = NewRandomString(28)
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.ClientId = ""
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.ClientId = NewId()
 	require.NotNil(t, ad.IsValid())
 
 	ad.UserId = NewRandomString(28)
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.UserId = ""
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.UserId = NewId()
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.Token = NewRandomString(22)
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.Token = NewId()
 	require.NotNil(t, ad.IsValid())
 
 	ad.RefreshToken = NewRandomString(28)
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.RefreshToken = NewId()
 	require.NotNil(t, ad.IsValid())
 
 	ad.RedirectUri = ""
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.RedirectUri = NewRandomString(28)
-	require.Error(t, ad.IsValid())
+	require.NotNil(t, ad.IsValid())
 
 	ad.RedirectUri = "http://example.com"
-	require.Error(t, ad.IsValid(), ad.IsValid())
+	require.Nil(t, ad.IsValid())
```

Comment:
This was false-passing before, as `Error`, but now that is checking for `Nil` is having the expected behavior.